frontend_app_gallery/src/component/CarouselArrow.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import { DIRECTION } from '../helper'
+import PropTypes from 'prop-types'
+
+const CarouselArrow = (props) => {
+  return (
+    <div
+      className={classnames(
+        'carousel__arrow',
+        'carousel__arrow__next',
+        props.direction === DIRECTION.RIGHT ? 'carousel__arrow__next' : 'carousel__arrow__prev'
```

Comment:
it looks like CarouselArrow get always rendered with classes: `.carousel__arrow .carousel__arrow__next .carousel__arrow__next` or `.carousel__arrow .carousel__arrow__next .carousel__arrow__prev`

also, it shouldn't have both `carousel__arrow` and `carousel__arrow__next`